assertk-common/src/main/kotlin/assertk/assertions/collection.kt

```diff
@@ -81,3 +82,17 @@ fun <T : Collection<*>> Assert<T>.containsAll(vararg elements: Any?) {
     val notFound = elements.filterNot { it in actual }
     expected("to contain all:${show(elements)} some elements were not found:${show(notFound)}")
 }
+
+/**
+ * Asserts the collection contains only the expected elements
+ * @see [containsNone]
+ * @see [containsExactly]
+ * @see [containsAll]
+ */
+fun <T : Collection<*>> Assert<T>.containsOnly(vararg elements: Any?) {
+    containsAll(*elements)
```

Comment:
Is this needed? Think it might be confusing if containsOnly shows the containsAll error message.

Reply:
I agree. I actually noticed this during testing and I couldn't tell which message was which. Might write the implementation for `containsAll`, or is there a better idea ?